resources/js/Shared/Components/SearchPanelSkeleton.vue

```diff
@@ -0,0 +1,100 @@
+          <div style="background-color: #26ccf0" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
+          <div style="background-color: #f5c604" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
+        </div>
+      </li>
+    </ul>
+  </div>
+
+  <div class="mx-auto w-3/4 animate-pulse p-4">
+    <ul role="list" class="divide-y divide-gray-300 ">
+      <li class="flex items-start justify-between gap-x-6 py-5">
+        <div class="flex flex-auto items-center">
+          <i class="br flat flag huge shrink-0" />
+          <div class="ml-4 flex flex-col justify-start">
+            <p class="mb-3 mr-2 h-2 w-24 rounded bg-gray-400 font-bold" />
+            <p class="h-2 w-12 rounded bg-blumilk-400 text-xs font-semibold" />
+          </div>
+        </div>
+        <div class="flex w-full flex-row-reverse flex-wrap items-center">
+          <div style="background-color: #f46c63" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
+        </div>
+      </li>
+    </ul>
+  </div>
+
+  <div class="mx-auto w-3/4 animate-pulse p-4">
+    <ul role="list" class="divide-y divide-gray-300 ">
+      <li class="flex items-start justify-between gap-x-6 py-5">
+        <div class="flex flex-auto items-center">
+          <i class="sc flat flag huge shrink-0" />
+          <div class="ml-4 flex flex-col justify-start">
+            <p class="mb-3 mr-2 h-2 w-24 rounded bg-gray-400 font-bold" />
+            <p class="h-2 w-12 rounded bg-blumilk-400 text-xs font-semibold" />
+          </div>
+        </div>
+        <div class="flex w-full flex-row-reverse flex-wrap items-center">
+          <div style="background-color: #80ff80" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
+          <div style="background-color: #34d186" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
+        </div>
+      </li>
+    </ul>
+  </div>
+
+  <div class="mx-auto w-3/4 animate-pulse p-4">
+    <ul role="list" class="divide-y divide-gray-300 ">
+      <li class="flex items-start justify-between gap-x-6 py-5">
+        <div class="flex flex-auto items-center">
+          <i class="gr flat flag huge shrink-0" />
+          <div class="ml-4 flex flex-col justify-start">
+            <p class="mb-3 mr-2 h-2 w-24 rounded bg-gray-400 font-bold" />
+            <p class="h-2 w-12 rounded bg-blumilk-400 text-xs font-semibold" />
+          </div>
+        </div>
+        <div class="flex w-full flex-row-reverse flex-wrap items-center">
+          <div style="background-color: #f5c604" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1 " />
```

Comment:
```suggestion
          <div style="background-color: #f5c604" class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1"/>
```

Please look for other occurrences extra spaces.